### DISCUSSION TITLE
Does minification actually work?

### DISCUSSION THREAD

**Discussion by floydjones1** [swc-project/swc#1492] [Q&A] [answered]
I tried swc-loader with webpack and I don't notice a difference with minification. If I disable minification on webpack I assume that the .swcrc config still would minify if that flag is set to true. I don't think that's the case

**Comment by kdy1** [comment, marked as answer]
API exists, but it's WIP

**Reply by floydjones1** [reply]
Okay thanks! Looking forward to it so I can speed up that aspect of my build as well !
